### PR TITLE
releases: add TiDB 3.0.19 release notes

### PR DESCRIPTION
**Pull request by uglyengineer**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/3952
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: releases/release-3.0.19.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## Bug 修复
+
++ TiDB
+
+    - 
+
++ TiKV
+
+    -
```
Comment (youjiali1995) [inline releases/release-3.0.19.md:25]:
```suggestion
    - 修复 TiKV 的 status server 解析响应出错导致 panic 的问题 [#8540](https://github.com/tikv/tikv/pull/8540)
```

File: releases/release-3.0.19.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+发版日期：2020 年 9 月 25 日
+
+TiDB 版本：3.0.19
+
+## 提升改进
+
++ TiDB
+
+    - 
+
```
Comment (youjiali1995) [inline releases/release-3.0.19.md:16]:
```suggestion
+ TiKV

    - 永久开启 `sync_log` [#8636](https://github.com/tikv/tikv/pull/8636)
```

File: releases/release-3.0.19.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
++ PD
+
+    - 
+
++ Tools
+
+    + Tools
```
Comment (3pointer) [inline releases/release-3.0.19.md:33]: - Lightning 
```suggestion
    + Tools
```
- 
```suggestion
    + Lightning
        - 修复了严格模式下 csv 中遇到不合法 UTF 字符集没有及时退出的问题 [#378](https://github.com/pingcap/tidb-lightning/pull/378)
```

File: releases/release-3.0.19.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+TiDB 版本：3.0.19
+
+## 兼容性变化
+
+
+## 提升改进
+
++ TiDB
+
+    -
```
Comment (wjhuang2016) [inline releases/release-3.0.19.md:18]: - 缓解故障恢复对 QPS 的影响 [#19764](https://github.com/pingcap/tidb/pull/19764)
- 支持调整 `union` 运算符的并发数 [#19885](https://github.com/pingcap/tidb/pull/19885)

File: releases/release-3.0.19.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
++ PD
+
+    -
+
+## Bug 修复
+
++ TiDB
+
+    -
```
Comment (wjhuang2016) [inline releases/release-3.0.19.md:32]: - 修复 `slow-log` 文件不存在导致查询出错的问题。 [#20050](https://github.com/pingcap/tidb/pull/20050)
    - 添加对 `SHOW STATS_META` 和 `SHOW STATS_BUCKET` 这两个命令的权限检查 [#19759](https://github.com/pingcap/tidb/pull/19759)
    - 禁止将 integer 的类型改成 decimal 的类型 [#19681](https://github.com/pingcap/tidb/pull/19681)
    - 修复 alter enum/set 没有检查一些限制的问题 [#20045](https://github.com/pingcap/tidb/pull/20045)
    - 修复 tidb-server 在 panic 后没有释放 table lock 的问题 [#20021](https://github.com/pingcap/tidb/pull/20021)
    - 修复 `OR` 运算符在 `WHERE` 子句中没有正常处理的问题 [#19901](https://github.com/pingcap/tidb/pull/19901)

File: releases/release-3.0.19.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+## Bug 修复
+
++ TiDB
+
+    - 修复 `slow-log` 文件不存在导致查询出错的问题。 [#20050](https://github.com/pingcap/tidb/pull/20050)
+    - 添加对 `SHOW STATS_META` 和 `SHOW STATS_BUCKET` 这两个命令的权限检查 [#19759](https://github.com/pingcap/tidb/pull/19759)
+    - 禁止将 integer 的类型改成 decimal 的类型 [#19681](https://github.com/pingcap/tidb/pull/19681)
+    - 修复 alter enum/set 没有检查一些限制的问题 [#20045](https://github.com/pingcap/tidb/pull/20045)
+    - 修复 tidb-server 在 panic 后没有释放 table lock 的问题 [#20021](https://github.com/pingcap/tidb/pull/20021)
+    - 修复 `OR` 运算符在 `WHERE` 子句中没有正常处理的问题 [#19901](https://github.com/pingcap/tidb/pull/19901)
```
Comment (wjhuang2016) [inline releases/release-3.0.19.md:38]: s/正常处理/正确处理

Comment (TomShawn) [inline releases/release-3.0.19.md:38]:
```suggestion
    - 修复 `OR` 运算符在 `WHERE` 子句中没有正确处理的问题 [#19901](https://github.com/pingcap/tidb/pull/19901)
```

File: releases/release-3.0.19.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+---
+
+# TiDB 3.0.19 Release Notes
+
+发版日期：2020 年 9 月 25 日
+
+TiDB 版本：3.0.19
+
+## 兼容性变化
+
```
Comment (nolouch) [inline releases/release-3.0.19.md:12]:
```suggestion
+ PD
       
       - 更改 PD 的 import path 从 `pingcap/pd` 迁移到 `tikv/pd` [#2779](https://github.com/pingcap/pd/pull/2779)
       - 更改 PD 的 copyright 信息，从 `PingCAP, Inc` 改为 `TiKV Project Authors` [#2777](https://github.com/pingcap/pd/pull/2777)
```

File: releases/release-3.0.19.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+    - 缓解故障恢复对 QPS 的影响 [#19764](https://github.com/pingcap/tidb/pull/19764)
+    - 支持调整 `union` 运算符的并发数 [#19885](https://github.com/pingcap/tidb/pull/19885)
+
++ TiKV
+
+    - 永久开启 `sync_log` [#8636](https://github.com/tikv/tikv/pull/8636)
+
++ PD
+
+    -
```
Comment (nolouch) [inline releases/release-3.0.19.md:27]:
```suggestion
    - 添加关于 PD 重启的告警规则 [#2789](https://github.com/pingcap/pd/pull/2789)
```

File: releases/release-3.0.19.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+## Bug 修复
+
++ TiDB
+
+    - 修复 `slow-log` 文件不存在导致查询出错的问题。 [#20050](https://github.com/pingcap/tidb/pull/20050)
```
Comment (TomShawn) [inline releases/release-3.0.19.md:33]:
```suggestion
    - 修复 `slow-log` 文件不存在导致查询出错的问题 [#20050](https://github.com/pingcap/tidb/pull/20050)
```

File: releases/release-3.0.19.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+    - 添加对 `SHOW STATS_META` 和 `SHOW STATS_BUCKET` 这两个命令的权限检查 [#19759](https://github.com/pingcap/tidb/pull/19759)
+    - 禁止将 integer 的类型改成 decimal 的类型 [#19681](https://github.com/pingcap/tidb/pull/19681)
```
Comment (TomShawn) [inline releases/release-3.0.19.md:35]:
```suggestion
    - 禁止将 Integer 类型改成 Decimal 类型 [#19681](https://github.com/pingcap/tidb/pull/19681)
```

File: releases/release-3.0.19.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+    - 修复 alter enum/set 没有检查一些限制的问题 [#20045](https://github.com/pingcap/tidb/pull/20045)
```
Comment (TomShawn) [inline releases/release-3.0.19.md:36]:
```suggestion
    - 修复更改 `ENUM`/`SET` 类型的列没有检查限制的问题 [#20045](https://github.com/pingcap/tidb/pull/20045)
```

File: releases/release-3.0.19.md [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+
++ TiKV
+
+    - 修复 TiKV 的 status server 解析响应出错导致 panic 的问题 [#8540](https://github.com/tikv/tikv/pull/8540)
+
++ Tools
+
+    + TiDB Lightning
+
+        - 修复了严格模式下 CSV 中遇到不合法 UTF 字符集没有及时退出的问题 [#378](https://github.com/pingcap/tidb-lightning/pull/378)
```
Comment (TomShawn) [inline releases/release-3.0.19.md:48]:
```suggestion
        - 修复了严格模式下 CSV 中遇到不合法 UTF 字符集没有及时退出进程的问题 [#378](https://github.com/pingcap/tidb-lightning/pull/378)
```

File: releases/release-3.0.19.md [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+## 提升改进
+
++ TiDB
+
+    - 缓解故障恢复对 QPS 的影响 [#19764](https://github.com/pingcap/tidb/pull/19764)
+    - 支持调整 `union` 运算符的并发数 [#19885](https://github.com/pingcap/tidb/pull/19885)
+
++ TiKV
+
+    - 永久开启 `sync_log` [#8636](https://github.com/tikv/tikv/pull/8636)
```
Comment (youjiali1995) [inline releases/release-3.0.19.md:27]:
```suggestion
    - 永久开启 `sync-log` [#8636](https://github.com/tikv/tikv/pull/8636)
```

File: releases/release-3.0.19.md [hunk 14]
```diff
@@ -0,0 +1,52 @@
...
+
+    - 添加关于 PD 重启的告警规则 [#2789](https://github.com/pingcap/pd/pull/2789)
+
+## Bug 修复
+
++ TiDB
+
+    - 修复 `slow-log` 文件不存在导致查询出错的问题 [#20050](https://github.com/pingcap/tidb/pull/20050)
+    - 添加对 `SHOW STATS_META` 和 `SHOW STATS_BUCKET` 这两个命令的权限检查 [#19759](https://github.com/pingcap/tidb/pull/19759)
+    - 禁止将 Integer 类型改成 Decimal 类型 [#19681](https://github.com/pingcap/tidb/pull/19681)
```
Comment (TomShawn) [inline releases/release-3.0.19.md:39]:
```suggestion
    - 禁止将 Decimal 类型改成 Integer 类型 [#19681](https://github.com/pingcap/tidb/pull/19681)
```

**Comment by TomShawn** [comment]
LGTM

**Review by wjhuang2016 — COMMENTED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]

**Pull request merged**